src/server/models/user.js

```diff
@@ -111,59 +127,70 @@ var UserSchema = new Schema({
   // Declares the name of the bus that the user can take
   availableBus: {
     type: String,
-    trim: true
+    trim: true,
+    displayName: 'Available Bus'
   },
   // Declares whether the user has noted they will be taking the bus
   bussing: {
     type: Boolean,
-    default: false
+    default: false,
+    displayName: 'Bussing'
   },
   // Declares the array of teammates that the user has defined
   teammates: [{
     type: String,
     match: [/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/,
-      'You must use a valid email']
+      'You must use a valid email'],
+    displayName: 'Teammates'
   }],
   // Declares the user's current application status
   // Rejected, Unconfirmed, Confirmed, Declined, Late, and Waitlisted
   status: {
     type: String,
-    trim: true
+    trim: true,
+    displayName: 'Status'
   },
   // Declares that the user has checked into the event on the day
   checkedIn: {
     type: Boolean,
-    default: false
+    default: false,
+    displayName: 'Checked In'
   },
   // Declares that the user's information is ready for sponsor consumption
   sanitized: {
     type: Boolean,
-    default: false
+    default: false,
+    displayName: 'Sanitized'
   },
   //Declares the user's provided race, required by events on an optional basis
   race: {
     type: String,
-    required: false
+    required: false,
+    displayName: 'Race'
```

Comment:
We should not have this public to organizers to sort by

Reply:
I personally don't even understand why we have it as a part of the registration? But if we do have it, and it can be exported, then why is it worse to have it here?

Reply:
I believe MLH asks hackathons to track it, which is why we added it for SD Hacks. I don't think we should allow organisers to sort by this because it would be... weird (unethical?) to allow it. I think we should silently track it when needed and not expose it as a UI field. 

Also, we don't export race in CSVs

Reply:
We let admin access every other field, including gender, as a way to verify all the applications and make changes if people request, without needing a developer to ever touch the database. Our website doesn't make any decisions about who is or isn't included in the events, it just holds the information for the organisers to do so. So if someone does decide to use this information unethically, that is their own decision separate from ours.

Reply:
If we don't export race in the CSV, and don't show it on the admin panel, why do we bother collecting it? Do we ever give it to MLH?

Reply:
We haven't given it to MLH afaik, I was told to add it as a feature before SD Hacks for us to be a member hackathon if I remember correctly.